classes/Validate.php

```diff
@@ -148,6 +148,22 @@ public static function isImageSize($size)
         return preg_match('/^[0-9]{1,4}$/', $size);
     }
 
+    /**
+     * Check whether given customer name is valid
+     *
+     * @param string $name Name to validate
+     *
+     * @return int 1 if given input is a name, 0 else
+     */
+    public static function isCustomerName($name)
```

Comment:
I did not think of it first but ... should'nt we update the license header to `2007-2019 PrestaShop` then ?

Reply:
yes we might need to :)